src/services/message.service.ts

```diff
@@ -31,6 +31,46 @@ export class MessageService {
     this._sendConstructedReport(report, { files: message.attachments.map((e) => e.url) });
   }
 
+  async sendMessage(
+    message: IMessage,
+    content: string,
+    reply: boolean,
+    argHeader?: string,
+    argFooter?: string,
+    argBreakChar?: string
```

Comment:
when requirements change for `sendMessage` it will be simpler to extend if you accept an optional object called `options` that contains the struct for additional parameters you can send to send message

Reply:
an additional requirement that would be nice in the future is to support embeds, accepting relevant embed parameters in this `options` object would be make the interface much simpler for consumers.

Reply:
a separate function for embed I believe. @tanndlin am I correct on that?

Reply:
Yeah, you can split it up into embeds then send it thru `messageService.sendPagedEmbed`
https://github.com/cs-discord-at-ucf/lion/blob/ff3e3a996aa603e78c4f5662a8d7616d98b29dde/src/services/message.service.ts#L42

Reply:
I have kinda added the "option" argument.  I had to make an interface for it since I could not figure out how to get it to work nicely with strings, without it bloating the code.  And the only sources I found on google were only for Java.  If you wanted it without an interface I would appreciate a link or example code.